### PR TITLE
Improved "Latest Recommendation" link in the document header

### PR DESCRIPTION
**Pull request by himorin**
***
<a href="https://pr-preview.s3.amazonaws.com/himorin/imsc/pull/537.html" title="Last updated on Apr 7, 2020, 4:32 PM UTC (1eecd1e)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/imsc/537/b847d80...himorin:1eecd1e.html" title="Last updated on Apr 7, 2020, 4:32 PM UTC (1eecd1e)">Diff</a>

### CODE REVIEW

File: imsc1/spec/ttml-ww-profiles.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
 <head>
   <meta charset='utf-8'>
 
-  <title>TTML Profiles for Internet Media Subtitles and Captions 1.0.1 (IMSC1)</title>
+  <title>TTML Profiles for Internet Media Subtitles and Captions 1.0.1 (IMSC1) (Link corrected 9 April 2020)</title>
```
Comment (nigelmegitt) [inline imsc1/spec/ttml-ww-profiles.html:7]: Not sure we need to add this change. @plehegar what is normal process here please?

File: imsc1/spec/ttml-ww-profiles.html [hunk 2]
```diff
@@ -21,7 +21,7 @@
                 ,   otherLinks: [{
                           key: 'Latest IMSC1 recommendation',
```
Comment (nigelmegitt) [inline imsc1/spec/ttml-ww-profiles.html:22]:
```suggestion
                          key: 'Latest IMSC recommendation',
```

File: imsc1/spec/ttml-ww-profiles.html (hunk 1 above)
Comment (palemieux) [reply imsc1/spec/ttml-ww-profiles.html:7]: The title should not be changed. A note should be sufficient -- see the following example.

https://www.w3.org/TR/2016/REC-ttml-imsc1-20160421/

File: imsc1/spec/ttml-ww-profiles.html [hunk 3]
```diff
@@ -84,6 +84,9 @@
   </section>
 
   <section id='sotd'>
+    <p class="note">
+      Status Update (April 2020): The link to the latest IMSC1 Recommendation in the header was fixed in-place on April 9 2020: they were pointing to TTML IMSC 1.0.1, and were fixed to link to the latest Recommendation of TTML IMSC.
```
Comment (palemieux) [inline imsc1/spec/ttml-ww-profiles.html:88]: `Status Update (9 April 2020):  In the document header, the "Latest IMSC1 Recommendation" link (pointing to the specific "TTML Profiles for Internet Media Subtitles and Captions 1.0.1" Recommendation) was replaced by a "Latest IMSC Recommendation" link pointing to the most recently published Recommendation in the "TTML Profiles for Internet Media Subtitles and Captions" family of Recommendations.`

Comment (himorin) [reply imsc1/spec/ttml-ww-profiles.html]: thank you for text. updated.

**Comment by himorin** [comment]
take down this PR, no way to use respec for in-place modification.

**Pull request closed without merging**